blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "Upgrades to the REPL in v0.6"
+author: <a href="https://kristofferc.github.io/">Kristoffer Carlsson</a>
+---
+
+Since v0.3, Julia comes with a REPL completely written in Julia ([PR #6270](https://github.com/JuliaLang/julia/pull/6270)).
```

Comment:
"Julia has come"

Maybe define "REPL", e.g.
```
Since version 0.3, Julia has come with a command-line interface — a [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) — that is completely written in Julia (....)
```